protocol/satellite-protocol/satellite/Event.proto

```diff
@@ -20,8 +20,7 @@ syntax = "proto3";
 
 package skywalking.v3;
 
-option go_package = "satellite/protocol";
-
+option go_package = "skywalking.apache.org/repo/goapi/satellite/event/v1";
```

Comment:
How does it work to just rename the package without recompile the proto files into go files?